pyocd/coresight/cortex_m.py

```diff
@@ -127,6 +127,9 @@ def register_name_to_index(reg):
             raise KeyError('cannot find %s core register' % reg)
     return reg
 
+def get_ipsr(context):
+    return context.readCoreRegister('xpsr') & 0x1ff
```

Comment:
`readCoreRegister` needs to be updated to `read_core_register`.

I'd also like to see a named constant for 0x1ff. For now it can go into `CortexM` (I'm probably going to extract the constants in `CortexM` into a constants module at some point).

Reply:
I'm tend to be of the school that if a constant is used in exactly one place, then you don't need a define for it, but will look neat if I add it alongside the other PSR masks.